vlm/src/main/scala/geotrellis/contrib/vlm/RasterSummary.scala

```diff
@@ -0,0 +1,100 @@
+package geotrellis.contrib.vlm
+
+import geotrellis.proj4.CRS
+import geotrellis.raster.{CellGrid, CellSize, CellType, RasterExtent}
+import geotrellis.spark._
+import geotrellis.spark.tiling.{LayoutDefinition, LayoutLevel, LayoutScheme}
+import geotrellis.util._
+import geotrellis.vector.{Extent, ProjectedExtent}
+import org.apache.spark.rdd.RDD
+
+case class RasterSummary(
+  crs: CRS,
+  cellType: CellType,
+  cellSize: CellSize,
+  extent: Extent,
+  cells: Long,
```

Comment:
This doesn't need to happen for this PR, but based on our discussion earlier, how would you feel about `RasterSummary` having both `GridBounds` and the number of non-`NoData` values in a given `RasterSource`?

Reply:
What's wrong with it? GridBounds are the actual bounds and number of NoData values is a `cells` number. This value is used to estimate better partition sizes as you remember from our conversation. At this point `cells` just computed a bit incorrect, see the comment below.

Reply:
Oh, I misspoke. I meant that it might be useful to have the number of all pixels and the number of non-`NoData` values

Reply:
Number of all pixels would be GridBounds size.

Reply:
Ah, that's true.